src/tools/windeployqt/main.cpp

```diff
@@ -1648,7 +1648,7 @@ int main(int argc, char **argv)
     const QByteArray qtBinPath = QFile::encodeName(QDir::toNativeSeparators(QCoreApplication::applicationDirPath()));
     QByteArray path = qgetenv("PATH");
     if (!path.contains(qtBinPath)) { // QTBUG-39177, ensure Qt is in the path so that qt.conf is taken into account.
-        path += ';';
+        path += QDir::listSeparator();
```

Comment:
Actually this is the BUG. THis bug fixes windeployqt under linux.

Please tell me how can I make the fix for Qt 5.15.x ? is it possible ?